src/jit/compiler.h

```diff
@@ -7503,6 +7503,21 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         return info.compCompHnd->isInSIMDModule(clsHnd);
     }
 
+    bool isIntrinsicType(CORINFO_CLASS_HANDLE clsHnd)
+    {
+        return (info.compCompHnd->getClassAttribs(clsHnd) & CORINFO_FLG_INTRINSIC_TYPE) != 0;
+    }
+
```

Comment:
I have mixed feelings about this, chances are we should not be calling `getClassAttribs` just to check one flag bit like this.

So I would remove this wrapper.

Reply:
On further review it looks like there are bigger issues to sort through here first, so leaving this as is is ok for now.

Reply:
Good point! I looked the JIT code, we can accumulate the call to `getClassAttribs` in importer.
I will refactor and remove the wrapper later.

Reply:
@AndyAyersMS could you clarify what you mean by "bigger issues to sort through"? Do you mean modifications to the jit to actually make use of this, or something else?